### DISCUSSION TITLE
requesting  confirmTransportationOptions return "ERROR: Sorry, something went wrong. Please try again, or refresh the page."

### DISCUSSION THREAD

**Discussion by priestName** [amzn/selling-partner-api-models#4462] [Q&A] [answered]
Always return the following error when requesting the confirmTransportationOptions interface
[{"code":"FBA_INB_0115","message":"ERROR: Sorry, something went wrong. Please try again, or refresh the page.","severity":"ERROR"}]
Have you encountered similar problems

**Comment by MetaMan2023** [comment]
> Always return the following error when requesting the confirmTransportationOptions interface [{"code":"FBA_INB_0115","message":"ERROR: Sorry, something went wrong. Please try again, or refresh the page.","severity":"ERROR"}] Can you provide specific reasons operationId: 4bd57508-8d9b-44da-b34d-1579dc76d1b5 820663a6-8ce8-45dd-b569-3b676460238d 7ee005b4-af63-4ded-9bff-3ce634a2c602

Is this via the API or the Seller Central interface?

And if Seller Central, you see that "raw error" like that on the screen?

**Reply by priestName** [reply]
This is a prompt returned through API request,When requesting repeatedly, it will still return "Sorry, something went wrong. Please try again, or refresh the page"

**Reply by MetaMan2023** [reply]
While I can't confirm because I don't know what data you are sending and formatting, etc.  I can say that this particular part of the API is working for us (following the left path, boxes packed up front) and we just successfully used that functionality the last few days.

Why again I can't confirm your specific issues I would check things like:

* Are you passing the right parameters / data and in the proper format -- JSON data ok?  Query data OK?

* Did you confirm the packing group first?    What about delivery windows (if a shipment type that needs delivery window)?   If you use APC UPS it doesn't need one which makes it a little easier.  Also were any errors thrown prior to that?

Maybe make the call to get the status of the inbound plan prior to making that call to see what the current status overall is and that the plan is valid up until that point.

**Reply by priestName** [reply]
Before confirmingTransportationOptions, I had already completed the following steps and returned them all as OperationStatus: SUCCESS
CreateInboundShipmentPlan
GeneratePackingOptions
ConfirmPackingOption
SetPackingInformation
GeneratePlacementOptions
GenerateTransportationOptions
ConfirmPlacementOption
Only the SetRackingInformation request has returned a warning message “WARNING: Box minimum cubic volume does not meet the expected minimum cubic cm”
However, when the final request for ConfirmTransportationOptions was made, getInboundOperationStatus returned a Status of FAILED with the prompt message "ERROR: Something went wrong. Please try again later."
Have you ever encountered this kind of problem message

**Reply by MetaMan2023** [reply]
> GeneratePlacementOptions GenerateTransportationOptions ConfirmPlacementOption

It sounds like you are calling:

ConfirmPlacementOption  AFTER you GenerateTransportationOptions

I think you need to call ConfirmPlacementOption BEFORE you call GenerateTransportationOptions?

**Reply by liudong19891218** [reply]
garbage amazon, I encountered the same problem, the most tragic thing is that the goods have been sent

**Reply by priestName** [reply]
> > GeneratePlacementOptions GenerateTransportationOptions ConfirmPlacementOption
> 
> It sounds like you are calling:
> 
> ConfirmPlacementOption AFTER you GenerateTransportationOptions
> 
> I think you need to call ConfirmPlacementOption BEFORE you call GenerateTransportationOptions?

After testing, the cause was identified, and it is possible that I used a non Amazon logistics option but did not confirm the delivery time using ConfirmDeliveryWindowOptions

**Reply by MetaMan2023** [reply, marked as answer]
Yes if you use a non APC, you must confirm delivery window options before confirming transportation options.

**Comment by MetaMan2023** [comment]
@priestName are you still having this issue?

We have been having it for weeks -- please see (and respond) to this thread in the "Issues" section:

https://github.com/amzn/selling-partner-api-models/issues/4656

**Reply by priestName** [reply]
thank you,We have already resolved this issue.
after confirming the delivery window using ConfirmDeliveryWindowOptions, we no longer have this issue.
We can only select non partner carriers when creating shipment confirmationTransportationOptions from the API. If we need to use Amazon Logistics, we can only go through the Send to Amazon workflow on the Amazon interface
